mina.gemspec

```diff
@@ -20,9 +20,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib', 'tasks']
 
   spec.add_dependency 'rake'
-  spec.add_dependency 'open4', '~> 1.3.4'
   spec.add_development_dependency 'rspec', '~> 3.5.0'
-  spec.add_development_dependency 'pry-byebug'
+  spec.add_development_dependency 'pry'
```

Comment:
Had to replace this because it's not supported on jruby (I think it's also a better option).